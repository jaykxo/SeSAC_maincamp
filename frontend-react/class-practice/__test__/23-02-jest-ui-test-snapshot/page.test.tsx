import JestUiTestPage from "@/app/section23/23-02-jest-ui-test-snapshot/page";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

it("내가 원하는대로 잘 그려지는지 확인하기", () => {
  const result = render(<JestUiTestPage />);
  expect(result.container).toMatchSnapshot();
});
