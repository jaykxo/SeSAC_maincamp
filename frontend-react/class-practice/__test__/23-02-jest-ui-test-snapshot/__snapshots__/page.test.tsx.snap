// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`내가 원하는대로 잘 그려지는지 확인하기 1`] = `
<div>
  <div>
    명수는 12살 입니다.
  </div>
  취미 입력: 
  <input
    type="text"
  />
  <button>
    등록하기
  </button>
</div>
`;
